popup/popup.html

```diff
@@ -11,13 +11,34 @@
     <div id="check_options" class="popup-font popup-info">
       To get live stream notifications, enter your API key and turn on the Live
       Stream Notifications option in the Add-On Options page.
+      <div class="option-text">
+        <label> API Key</label>
+        <div class="option-infobutton-container">
+          <img src="../options/infobutton.png"/>
+          <div class="option-infobutton-text">
+            This extension uses the Giant Bomb API. This API requires
+            the use of an API key, and the extension has a default key. However, the
+            API has usage rate limits per API key, so it is recommended that you provide
+            your own for better reliability. You can find your key by logging in to
+            Giant Bomb and navigating to <a href="http://www.giantbomb.com/api" target="_blank">giantbomb.com/api</a>.
+            Your key will never be logged or shared by Giant Bomb QoL.
+          </div>
+        </div>
+      </div>
+      <div class="centered">
+        <input type="text" id="text_api_key" placeholder="Default Key (Not Recommended)"/><button type="button" id="submit_key">Activate</button><br>
```

Comment:
As for the Activate button, I'd prefer that being removed, and the popup automatically checking the length of the API key on input. If it's 40 characters, try the chats API endpoint. If it gets a good result, repopulate the popup with the result and save the key. If the result is bad, or if the key isn't 40 characters, put a red border on the input.

This is a pretty involved change so it's not necessary for me to approve the pull request, just an idea I had.

Reply:
I can do that.  I've been unable to find the API documentation, which would be immensely helpful in solving this.  I'll fix this before the next pull request.

Reply:
Would it be helpful to have a loading icon while we're hitting the API?  It's asynchronous by nature, so that might give the user an idea of the application status.

Reply:
I can see this project requiring that function in the future, so it may be a good idea to separate it into its own file.  Problem is, that's one more HTTP request.  Think it's worth it?